assets/src/scss/pages/post/_article-content.scss

```diff
@@ -216,22 +216,27 @@
       font-family: var(--font-family-heading);
       padding-top: 0;
       padding-bottom: 0;
+      line-clamp: 2;
       -webkit-line-clamp: 2;
     }
 
     .article-list-item-content {
       width: 100%;
+      line-clamp: 3;
       -webkit-line-clamp: 3;
 
       @include medium-and-up {
+        line-clamp: 2;
         -webkit-line-clamp: 2;
       }
 
       @include large-and-up {
+        line-clamp: 3;
         -webkit-line-clamp: 3;
       }
 
       @include x-large-and-up {
```

Comment:
I left a comment here on [my commit](https://github.com/greenpeace/planet4-master-theme/pull/2188/commits/8c168305b03ade75dd2e5f801ddaded3ec94bf79#r1723775030). What do you think?

Reply:
Yes, that makes sense! Updated 🙂